sdp/sdp.go

```diff
@@ -191,3 +193,512 @@ func CodecsFromMediaDescription(m *sdp.MediaDescription) (out []sdp.Codec, err e
 
 	return out, nil
 }
+
+func GetBundleMid(parsed *sdp.SessionDescription) (string, bool) {
```

Comment:
there's a lot of new code here, would any of it benefit from tests?

Reply:
Yes, will write them next.